README.md

```diff
@@ -200,11 +200,40 @@ For a working example on DigitalOcean using [`external-dns`](https://github.com/
     [open|xdg-open] http://localhost:8004
     ```
 
+*EFK stack for logging*
+
+* [`elasticsearch`](https://github.com/elastic/helm-charts/tree/master/elasticsearch) is a distributed, RESTful search and analytics engine and it's is used for log storage
+    ```bash
+    kubectl port-forward service/elasticsearch-master 9200:9200 -n observe
+    [open|xdg-open] http://localhost:9200
```

Comment:
remove the open command, having the port forward is enough

Reply:
I have the same also above, do you think that I should remove all of them?

Reply:
Yeah, I would 👍

Reply:
done